packages/react/src/Header/Header.tsx

```diff
@@ -64,31 +76,19 @@ const HeaderRoot = forwardRef(
 
     return (
       <header {...restProps} className={clsx('ams-header', className)} ref={ref}>
-        <div className="ams-header__branding">
-          <a className="ams-header__logo-link" href={logoLink}>
-            <span className="ams-visually-hidden">{logoLinkTitle}</span>
-            <Logo brand={logoBrand} />
-          </a>
-          {brandName && (
-            <Heading className="ams-header__brand-name" level={1} size="level-5">
-              {brandName}
-            </Heading>
-          )}
-        </div>
+        <a className="ams-header__logo-link" href={logoLink}>
+          <span className="ams-visually-hidden">{logoLinkTitle}</span>
+          <LogoLinkContent brandName={brandName} logoBrand={logoBrand} />
+        </a>
         {(children || menuItems) && (
           <nav aria-labelledby="primary-navigation" className="ams-header__navigation">
             <h2 className="ams-visually-hidden" id="primary-navigation">
               {navigationLabel}
             </h2>
```

Comment:
Do we want to keep this `h2` here? That would mean the `h2` comes before the `h1` on the page. That's allowed in HTML, but maybe it's a little confusing?

Gov.uk does do it, but I can't find any other examples of this. Most examples I found use `aria-label`

Reply:
As you mentioned today, assistive tech users can navigate by headings or landmarks. Would they want to come across the navigation if they navigate by headings? I would tend to think not, given that they have the choice. So `aria-label`?